src/libMessage/Messenger.cpp

```diff
@@ -265,7 +265,9 @@ bool ProtobufToAccountDelta(const ProtoAccount& protoAccount, Account& account,
   ProtobufByteArrayToNumber<uint128_t, UINT128_SIZE>(protoAccount.balance(),
                                                      tmpNumber);
 
-  int256_t balanceDelta = protoAccount.numbersign() ? tmpNumber : 0 - tmpNumber;
+  int256_t balanceDelta = protoAccount.numbersign()
```

Comment:
Should we change to `int128_t`?  Also perhaps `Account::ChangeBalance` (currently takes in `int256_t`)?

Reply:
Ah sorry, the recommendation is to keep 256